tango/workspace_server.py

```diff
@@ -1,20 +1,52 @@
 import logging
-from http.server import ThreadingHTTPServer, BaseHTTPRequestHandler
+from http.server import ThreadingHTTPServer, BaseHTTPRequestHandler, SimpleHTTPRequestHandler
```

Comment:
Is the goal to only depend on stdlib?

I'm used to Python web servers written at AI2 using Flask, is all.

Reply:
The goal is that we can spin up a "throwaway" web server while the main process runs the experiment. We launch this with `server = WorkspaceServer(address, workspace); server.serve_in_background();`, and then the rest of the code goes on modifying `workspace` while it runs.

Reply:
@dirkgr 👆

Reply:
That makes sense. I guess I'm surprised to see the base HTTP server constructs used instead of a Flask, which gives a nice syntax for routing and such. (I think you can still use it and climb port numbers until you arrive at a working port.)